### ISSUE TITLE
idpResponse.toIntent() in sample app

### ISSUE THREAD

**Issue by halfbaked** [firebase/FirebaseUI-Android#833]
The following line

https://github.com/firebase/FirebaseUI-Android/blob/master/app/src/main/java/com/firebase/uidemo/auth/SignedInActivity.java#L86

only works because the sample app has the the same package stem com.firebase. 
The average Joe trying to use the sample app as a template for getting FirebaseUI working in his own app will have a different package name, and it will fail for him.

**Comment by samtstern** [comment]
@halfbaked good catch!  Instead we should be adding the `IdpResponse` to the Intent as a Parcelable:

Here's all toIntent does:
```
return new Intent().putExtra(ExtraConstants.EXTRA_IDP_RESPONSE, this);
```

**Comment by samtstern** [comment]
The fix for this issue was released in version `2.2.0`.

**Comment by halfbaked** [comment]
You guys don't hang around. Good work!
